doc/source/theory/FlavorSpace.rst

```diff
@@ -75,31 +76,126 @@ The mapping between the Evolution Basis and the +/- Basis is given by
 - this basis is *not* normalized with respect to the canonical Flavor Basis
 - the basis transformation from the Flavor Basis is implemented in
   :data:`~eko.basis_rotation.rotate_flavor_to_evolution`
+- the photon is just a spectator and does not couple to anyone
 
-Intrinsic Evolution Bases
--------------------------
+Intrinsic QCD Evolution Bases
+-----------------------------
 
-However, the Evolution Basis is not yet the most decoupled basis if we consider intrinsic evolution.
+However, the QCD Evolution Basis is not yet the most decoupled basis if we consider intrinsic evolution.
 The intrinsic distributions do *not* participate in the |DGLAP| equation but instead evolve with a unity operator:
 this makes, e.g. :math:`T_{15}` a composite object in a evolution range below the charm mass.
 Instead, we will keep the non participating distributions here in their :math:`q^\pm` representation.
-The Intrinsic Evolution Bases will explicitly depend on the number of light flavors :math:`n_f`.
+The Intrinsic QCD Evolution Bases will explicitly depend on the number of light flavors :math:`n_f`.
 For :math:`n_f=3` we define (the other cases are defined analogously):
 
 .. math ::
-    \mathcal F \sim  \mathcal F_{iev,3} = \span(g, \Sigma_{(3)}, V_{(3)}, V_3, V_8, T_3, T_8, c^+, c^-, b^+, b^-, t^+, t^-)
+  \Sigma_{(3)} &= u^+ + d^+ +s^+\\
+  V_{(3)} = u^- + d^- + s^-\\
+  \mathcal F \sim  \mathcal F_{iev,3} &= \span(\gamma, g, \Sigma_{(3)}, V_{(3)}, V_3, V_8, T_3, T_8, c^+, c^-, b^+, b^-, t^+, t^-)
 
-where we defined :math:`\Sigma_{(3)} = \sum\limits_{j=1}^3 q_j^+` and :math:`V_{(3)} = \sum\limits_{j=1}^3 q_j^-`
-(not to be confused with the usual :math:`V_3`).
+where :math:`V_{(3)}` is to be confused with the usual (QCD like) :math:`V_3`.
 
-- for :math:`n_f=6` the Intrinsic Evolution Basis coincides with the Evolution Basis: :math:`\mathcal F_{iev,6} = \mathcal F_{ev}`
+- for :math:`n_f=6` the Intrinsic QCD Evolution Basis coincides with the QCD Evolution Basis: :math:`\mathcal F_{iev,6} = \mathcal F_{ev}`
 - this basis is *not* normalized with respect to the canonical Flavor Basis
 - the basis transformation from the Flavor Basis is implemented in
   :meth:`~eko.evolution_operator.flavors.pids_from_intrinsic_evol`
 - note that for the case of non-intrinsic component the higher elements in :math:`\mathcal F_{ev}` do become linear dependent
   to other basis vectors (e.g. :math:`\left. T_{15}\right|_{c^+ = 0} = \Sigma`) but are non zero - instead in :math:`\mathcal F_{iev,3}`
   this direction vanishes
+- the photon is just a spectator and does not couple to anyone
 
+
+QED :math:`\otimes` QCD Evolution Basis
```

Comment:
I wonder if it will be difficult to reference this heading in Sphinx with this name (and especially the `:math:`) - maybe we should rename to "Unified Evolution Basis"? same below ...

Reply:
You can add custom targets to reference headings (like `\label{}` in LaTeX), the only thing I'm not sure about is if it's a problem  rendering the link content. Maybe it's just fine.